tests/security/security_smoke.py

```
@@ -114,15 +114,31 @@ def check_xss_payload_not_reflected(base_url: str) -> CheckResult:
+    """
+    session = requests.Session()
+    # Sub-megabyte body keeps the probe fast on slow CI links while still stressing parsers.
+    big_value = "A" * (768 * 1024)
     try:
-        r = requests.post(
+        csrf_r = session.get(f"{base_url}/api/auth/csrf/", timeout=10)
+        csrf_headers = {}
+        if csrf_r.status_code == 200:
+            try:
+                token = csrf_r.json().get("csrfToken", "")
+            except Exception:
```

Ruff (0.15.9)

[warning] Do not catch blind exception: `Exception`

(BLE001)